internal/provider/resource_image_repo.go

```diff
@@ -89,6 +90,10 @@ func (r *imageRepoResource) Schema(_ context.Context, _ resource.SchemaRequest,
 					listvalidator.ValueStringsAre(validators.ValidateStringFuncs(validBundlesValue)),
 				},
 			},
+			"readme": schema.StringAttribute{
+				Description: "The README for this repo.",
```

Comment:
Do we expect the READMEs to always be valid markdown? A total nerd snipe, but it may be fun/useful to write a validator to parse the README and check if it's valid.

Reply:
im not sure best way to determine whether something is valid markdown. I'd like to defer any client-side validation of this field for now

Reply:
Yeah no worries, I poked around real quick and didn't see any obvious package that would accomplish this easily.